src/services/productService.js

```diff
@@ -21,6 +21,20 @@ const getProductList = async (
   return campList;
 };
 
+const getCampById = async (campId) => {
+  try {
+    const existCamp = await productDao.checkCampById(campId);
+
+    if (existCamp) return await productDao.getCampById(campId);
```

Comment:
다음의 로직이 한 끝 차이로 수정한 결과이지만 좀 더 다른 분들과 다른 코드들의 문체와 비슷해보일 것 같네요!

```suggestion
    const existCamp = await productDao.checkCampById(campId);

    if (!existCamp) throw error ... 이하 중략
    
    return await productDao.getCampById(campId);
```